dkms.in

```diff
@@ -981,6 +981,7 @@ prepare_signing()
     # scripts/sign-file.c in kernel source also supports using "pkcs11:..." as private key
     if [[ "$mok_signing_key" != "pkcs11:"* ]] && ( [ ! -f "$mok_signing_key" ] || [ ! -f "$mok_certificate" ] ); then
         echo "Certificate or key are missing, generating self signed certificate for MOK..."
+        command -v openssl >/dev/null || echo "openssl not found, can't generate key and certificate."
```

Comment:
Nit: Please also add a "return" in this case.

Can you rebase the series against master and look at the CI failures. Having the `BUILD_EXCLUSIVE` combinations checked (as mentioned earlier) would be great, but can be done as follow-up.

Thanks again o/